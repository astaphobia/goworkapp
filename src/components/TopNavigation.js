import React from 'react'
import TopLogo from '@/../images/top-logo.png'
import { Link } from 'react-router-dom'

const TopNavigation = () => {
    return (
        <nav className="navbar navbar-expand-lg navbar-light bg-danger">
            <div className="container">
                <Link to={'/'} className="navbar-brand">
                    <img src={TopLogo} width="120px" />
                </Link>
                <button className="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
                    <span className="navbar-toggler-icon"></span>
                </button>

                <div className="collapse navbar-collapse" id="navbarSupportedContent">
                    <ul className="navbar-nav mr-auto">
                        <li className="nav-item active">
                            <Link to={'/'} className="nav-link">
                                Keseluruhan Kerja
                            </Link>
                        </li>
                    </ul>
                </div>
            </div>
        </nav>
    )
}
export default TopNavigation