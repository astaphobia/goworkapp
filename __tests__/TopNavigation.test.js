import React from 'react'
import { shallow } from 'enzyme'

import TopNavigation from '@/components/TopNavigation'

describe('TopNav', () => {
    it('hope doing well TopNav', () => {
        const comp = shallow(<TopNavigation />)
        expect(comp).toMatchSnapshot()
    })
})