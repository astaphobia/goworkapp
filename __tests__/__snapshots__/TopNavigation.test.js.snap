// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TopNav hope doing well TopNav 1`] = `
<nav
  className="navbar navbar-expand-lg navbar-light bg-danger"
>
  <div
    className="container"
  >
    <Link
      className="navbar-brand"
      replace={false}
      to="/"
    >
      <img
        src="test-file-stub"
        width="120px"
      />
    </Link>
    <button
      aria-controls="navbarSupportedContent"
      aria-expanded="false"
      aria-label="Toggle navigation"
      className="navbar-toggler"
      data-target="#navbarSupportedContent"
      data-toggle="collapse"
      type="button"
    >
      <span
        className="navbar-toggler-icon"
      />
    </button>
    <div
      className="collapse navbar-collapse"
      id="navbarSupportedContent"
    >
      <ul
        className="navbar-nav mr-auto"
      >
        <li
          className="nav-item active"
        >
          <Link
            className="nav-link"
            replace={false}
            to="/"
          >
            Keseluruhan Kerja
          </Link>
        </li>
      </ul>
    </div>
  </div>
</nav>
`;
